.github/workflows/autopublish.yml

```
@@ -0,0 +1,56 @@
+name: Faceted Search Auto Publish
+
+on:
+  push:
+    branches: [master, develop, main]
```

We don't have a master branch on this repository
Let's give a try on gitflow there and not publish a release on develop

```suggestion
    branches: [main]
```